onnx/defs/operator_sets.h

```diff
@@ -942,6 +942,7 @@ class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 14, Sub);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 14, Mul);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 14, Div);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 14, Identity);
+class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 14, GridSampler);
```

Comment:
you should add this to opset 15 as this is the latest version

Reply:
Now exporter only support opset 14 after this PR merged. https://github.com/pytorch/pytorch/pull/59486.
So I'm not sure if it's better to support exporter to community or 1-p customer in opset14, or it's needed to use the latest version for ONNX？

Reply:
opset 14 is already released and you cannot add a new op to a released opset. Work for opset15 is in progress now and any op update or new op should be added to opset 15. Once onnx releases (next one is scheduled for July end) exporter can add support for this new opset.